x/profiles/types/models_chain_links_test.go

```diff
@@ -306,6 +329,65 @@ func TestBase58Address_GetValue(t *testing.T) {
 
 // --------------------------------------------------------------------------------------------------------------------
 
+func TestHexAddress_Validate(t *testing.T) {
+	testCases := []struct {
+		name      string
+		address   *types.HexAddress
+		shouldErr bool
+	}{
+		{
+			name:      "empty address returns error",
+			address:   types.NewHexAddress("", ""),
```

Comment:
We can cover two cases in one: empty and blank
```suggestion
			address:   types.NewHexAddress("", "   "),
```